README.md

```diff
@@ -137,6 +137,37 @@ Give FUSE permission to run as stated in the instructions and you should be good
 | [BLAKE2](https://github.com/BLAKE2/libb2)                | 2018-07 | CC0           |
 | [whereami](https://github.com/gpakosz/whereami)          | 2018-09 | WTFPLV2       |
 
+# Configuring wake
+
+Certain characteristics of wake execution can be configured for all invocations
+of the wake tool. For example, the repo may set the minimum wake version or a
+user may set the verbosity of a log message. This is achieved  via two config
+files: .wakeroot and the user config. The user config overrides .wakeroot if
+two values conflict. Certain values may only be specified in a specific
+location. For example, min version may only be set in the .wakeroot. Both files
+contain JSON5 source where the root object may contain the following keys.
+
+| Key | Description | Required | Type | .wakeroot allowed | user config allowed   | Default |
+| --- | ----------- | -------- | ---- | ----------------- | --------------------- | ------- |
+| version | SemVer compatible  with the repo | N  | SemVer string | Y | N | "" |
+| user_config | Path to user config for this repo. Allows a different config for each repo. | N | shell expandable path string | Y | N | $XDG_CONFIG_HOME/.wake.json if $XDG_CONFIG_HOME set, $HOME/.config/.wake.json otherwise |
```

Comment:
the wake.json file should not be hidden. config files under .config are never hidden in my experience. .config is already hidden and that's enough. Once you're in the directory you want to see those files. I must have missed this in the default function above but please fix it there.